src/modeler/AutoRest.Swagger/Validation/SecurityDefinitionsStructureValidation.cs

```diff
@@ -24,6 +24,11 @@ public class SecurityDefinitionsStructureValidation : TypedRule<Dictionary<strin
         public override ValidationCategory ValidationCategory => ValidationCategory.SDKViolation;
 
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public override ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.ServiceImpactingChanges;
```

Comment:
`securityDefinitions` don't affect our code generation as far as I can tell, think it's mainly documentational or to make the Swagger "complete" in the sense of documenting how to access the service. At least for C#, it seems like the same credential mechanics are generated regardless.